sdk/tables/Azure.Data.Tables/src/TableRestClient.cs

```diff
@@ -19,6 +21,7 @@ internal partial class TableRestClient
         internal ClientDiagnostics clientDiagnostics => _clientDiagnostics;
         internal string endpoint => url;
         internal string clientVersion => version;
+        private static readonly Regex s_entityIndexRegex = new Regex(@"""value"":""(?<index>[\d]+):", RegexOptions.Compiled);
```

Comment:
Should we parse JSON instead?

Reply:
In this case it would only get me closer to the `value` property which I'd then have to parse with SubString. Do you see other benefits?

Reply:
Ah, I didn't realize that the property itself isn't just the index.